js/dynamics.js

```diff
@@ -0,0 +1,17 @@
+const menuIcon = document.querySelector('#humberger_menu_icon');
+const menuList = document.querySelector('#humberger_menu_list');
+const closeMenuIcon = document.querySelector('#close-menu');
+const logoHeader = document.querySelector('.logo h1');
+menuIcon.addEventListener('click', () => {
+  closeMenuIcon.classList.add('show-icon');
+  menuList.classList.add('show');
+  menuIcon.classList.add('hide-icon');
+  logoHeader.classList.add('hide-icon');
+});
```

Comment:
Hello @mnekx, you have indeed done a great job with the mobile menu feature and I have checked and most of the requirements have been met 💪🏻! However, I have noticed that when the user clicks (or taps) any option of the mobile menu, the mobile menu does not disappear.

- [ ] According to the project requirements, the menu is meant to disappear when an option is selected. Please kindly fix this behavior.